packages/generators/utils/entry.ts

```diff
@@ -18,19 +18,18 @@ interface CustomGenerator extends Generator {
 
 export default function entry(
 	self: CustomGenerator,
-	answer: {
-		entryType: boolean;
-	}
+	multiEntries: boolean,
 ): Promise<void | {}> {
 	let entryIdentifiers: string[];
 	let result: Promise<void | {}>;
-	if (answer.entryType) {
+	if (multiEntries) {
 		result = self
 			.prompt([
 				InputValidate(
 					"multipleEntries",
-					"Type the names you want for your modules (entry files), separated by comma [example: app,vendor]",
-					validate
+					"How do you want to name your bundles? (separated by comma)",
+					validate,
+					"pageOne, pageTwo"
```

Comment:
I'd CC @addyosmani here (sorry) for a help on how to name the default name of multiple bundles in webpack. 

Config will look like this: 

```js
{
pageOne: './foo/bar',
pageTwo: './baz'
}
```

What naming should be instead of `PageOne` and `PageTwo` to give the user an indication of their entry points? I'm suggesting that we have `pageEntryOne` instead to explicitly tell the user that this is an entry in terms of bundles

Reply:
Those names are taken from the [documentation](https://github.com/misterdev/webpack/blob/output-reporter/lib/ReporterPlugin.js) where there are 2 examples, one with `app, adminApp` and one (more generic) with `pageOne, pageTwo, pageThree`.

Since multiple bundles are a more "advanced" option that you have select before using, I expect the majority of people to be aware of what they are doing. Also, someone looking for information may end up in the documentation and find a consistent example.

IMO if we want to move on we can keep those and then change them if we find something better. What do you think @evenstensberg? :)

Reply:
aye let's keep it like this